commit d87c0946496066765e158284fb4ed7c954954848 (refs/changes/46/3646/2)
Author: Brian Bennett <brian.bennett@joyent.com>
Date:   2018-06-21T13:32:09-07:00 (1 year, 4 months ago)
    
    joyent/node-manta#343 Want `mjob wait` as alias for `mjob watch`

diff --git a/bin/mjob b/bin/mjob
index b8196de..c3bfe4f 100755
--- a/bin/mjob
+++ b/bin/mjob
@@ -1,7 +1,7 @@
 #!/usr/bin/env node
 // -*- mode: js -*-
 /*
- * Copyright 2016 Joyent, Inc.
+ * Copyright 2018 Joyent, Inc.
  */
 
 var EventEmitter = require('events').EventEmitter;
@@ -1364,6 +1364,7 @@ MJob.prototype.do_watch.help = (
         '{{options}}'
 );
 
+MJob.prototype.do_watch.aliases = ['wait'];
 
 
 //-- Close
